scalableminds/webknossos
# Add measurement tools for trees in tree tab

### URL of deployed dev instance (used for testing):
- https://measurepathlength.webknossos.xyz

### Steps to test:
- create a skeleton and use the dropdown in the tree-list-tab to measure the length of the current skeleton or of all skeletons
- double-check the calculated length (e.g., using the scalebar)

### Issues:
- fixes #4842 

------
